.travis.yml

```diff
@@ -4,9 +4,28 @@ node_js:
   - '5'
 env:
   - CXX=g++-4.8
+
+services:
+  - docker
+
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
     packages:
       - g++-4.8
+
+env:
+  - global:
+    - COMMIT=${TRAVIS_COMMIT::8}
+    - SANITIZED_BRANCH=$(echo $TRAVIS_BRANCH|sed 's|/|-|g')
+    - REPO=sheltertechsf/askdarcel-web
+
+after_success:
```

Comment:
Sometime soon I'm going to move this script to some shared location, since it's used both here and in the API repository.